.github/workflows/release_update_docs_px_dev.yaml

```diff
@@ -0,0 +1,105 @@
+---
+name: update-docs-px-dev-on-release
+on:
+  push:
+    # TODO(ddelnano): Remove this in favor of vizier release events only once testing is complete
+    branches:
+    - test-create-docs-px-dev-prs-on-documentation-updates
+  release:
+    types: [published]
+permissions:
+  contents: read
+jobs:
+  # TODO(ddelnano): Uncomment this and retest after job is verified fully
+  # if: startsWith(github.head_ref, 'release/vizier/')
```

Comment:
the `if` should be inside the job

Reply:
Yea, I noticed that when I was prepping the final change for this PR. However, I still need to test that the job's previous functionality works after the most recent refactor.

Reply:
This will be tested in my most recent update. I moved it into the `get-dev-image` job since all of the other ones depend on it.